controllers/gce/firewalls/interfaces.go

```diff
@@ -36,5 +36,7 @@ type Firewall interface {
 	DeleteFirewall(name string) error
 	UpdateFirewall(f *compute.Firewall) error
 	GetNodeTags(nodeNames []string) ([]string, error)
+	NetworkProjectID() string
 	NetworkURL() string
+	OnXPN() bool
```

Comment:
Add a comment saying this will return true if XPN is enabled in GCP project.
optional: Probably also link to some XPN documentation?